HACKING.md

```diff
@@ -81,15 +52,28 @@ e.g.; to update the list for 16.04,
 
 ### Installing using pip
 
+If you don't have python3 installed, you can use the one from the archives:
+
+    sudo apt install python3-pip python3-setuptools python3-pkg-resources
+
 Install the needed dependencies.
 
-    sudo apt install build-essential python3-dev libapt-pkg-dev libsodium-dev gcc libffi-dev
+    sudo apt install build-essential python3-dev libapt-pkg-dev libsodium-dev gcc libffi-dev libarchive13
 
 If installing to `PYTHONHOME` run:
 
-    pip install --user -r requirements.txt -r requirements-devel.txt
+    pip3 install --user -r requirements.txt -r requirements-devel.txt
 
 If your prefer installing in a virtualenv, then in an activated environment run:
 
-    pip install -r requirements.txt -r requirements-devel.txt
+    pip3 install -r requirements.txt -r requirements-devel.txt
+
+### Testing
+
+We assume you have run through the installation instructions, to run all the tests execute:
+
+    ./runtests.sh
+
+You can selectively run a selective group of tests like:
 
+    ./runtests.sh [static|unit|integration|snaps]
```

Comment:
This will actually try to set up an all snaps image with ubuntu-device-flash. I think that the right thing to do now is to run the snaps tests locally:
https://github.com/snapcore/snapcraft/pull/911